recipe/meta.yaml

```diff
@@ -120,6 +121,7 @@ requirements:
   - curl
   - eigen
   - embree
+  - ffmpeg>=3.4.1
```

Comment:
It's possible that the change only needs to be made in the run section with a `pin_compatable` function. Did you upload the tarball for this already?

Reply:
not to the USGS channel because of this error, I wasn't 100% sure whether to add to one or both so I just did both to be safe. Not super knowledgable of the pin_compatible stuff, what would be the benefit here?

Reply:
The main benefit is that it wouldn't pin ffmpeg in two places, we would only need to deal with a single pin. It's also possible that we are getting some version for the build and a different version in run, I don't really understand why conda does this but it happened in the past which is why there are `pin_compatable` definitions. There is a file you can look at in the build that defines what packages were obtained at the different steps from the build which will inform you whether we should use the `pin_compatable` vs a hard pin in both `build` and `run`

Reply:
The file to check the versions obtained is under `info/recipe/meta.yaml`

Reply:
made the change, I'm doing another linux build to test it.

Reply:
ubuntu build cam back fine so I think this is good to go